tests/unit/components/message/Message.spec.jsx

```diff
@@ -0,0 +1,55 @@
+import { render, screen } from '@testing-library/react'
+import { BrowserRouter as Router } from 'react-router-dom'
+
+import someAvatar from '~/assets/img/student-home/bag.png'
+import { UserRoleEnum } from '~/types'
+import { getFormatedDate } from '~/utils/helper-functions'
+
+import Message from '~/components/message/Message'
+
+describe('Message component', () => {
+  const newAuthor = {
+    _id: '1234',
+    firstName: 'Kyle',
+    lastName: 'Jason',
+    photo: someAvatar,
+    createdAt: new Date()
+  }
+
+  const mockMessage = {
+    _id: 'newmess1',
+    author: newAuthor,
+    authorRole: UserRoleEnum.Student,
+    messageContent: 'how about some bruh moment'
+  }
+
+  it('should render the author name and message content', () => {
+    render(
+      <Router>
+        <Message message={mockMessage} />
+      </Router>
+    )
```

Comment:
Move it to beforeEach, because you use it in two places.

Reply:
fixed it